server/game/cards/02_SHD/leaders/BobaFettDaimyo.ts

```diff
@@ -0,0 +1,43 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { LeaderUnitCard } from '../../../core/card/LeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class BobaFettDaimyo extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9334480612',
+            internalName: 'boba-fett#daimyo',
+        };
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addTriggeredAbility({
+            title: 'When you play a unit that has 1 or more keywords: You may exhaust this leader. If you do, give a friendly unit +1/+0 for this phase.',
```

Comment:
Typically the ability is just the description of the effect, we leave out the trigger and the cost. So in this case I would just do `Give a friendly unit +1/+0 for this phase` and then remove the `activePromptTitle` from the target resolver.

We might change that way of doing things once the FE is up but for now let's just keep everything consistent.